handlers/handlers.go

```diff
@@ -245,14 +236,6 @@ func filterableLanding(w http.ResponseWriter, req *http.Request, dc DatasetClien
 		return
 	}
 
-	var bc []zebedee.Breadcrumb
```

Comment:
Note: I moved GetBreadcrumbs to a later point in the function, so that we only do the call if it's necessary